pkg/rollout/rollout.go

```diff
@@ -122,7 +122,12 @@ func (r *Rollout) UpdateService(svc *run.Service) (*run.Service, error) {
 	if isNewCandidate(svc, candidate) {
 		r.log.Debug("new candidate, assign some traffic")
 		svc = r.PrepareRollForward(svc, stable, candidate)
-		svc = r.updateAnnotations(svc, stable, candidate, "new candidate, no health report available yet")
+		svc = r.updateAnnotations(svc, stable, candidate)
+
+		// TODO(gvso): create a setHealthReportAnnotation that appends the
+		// current time to the report before setting the annotation.
```

Comment:
this probably could be part of the health report IMO

Reply:
My initial thought was also this one, but the problem is that we don't generate a report when we have a new candidate. We have a string literal in that case